_data/projects/blitzed-irc-trivia.yml

```diff
@@ -0,0 +1,14 @@
+name: Blitzed IRC Trivia
+desc: Blitzed IRC Trivia is an IRC bot that will connect to a server, join a channel, and ask trivia questions. Allows team or single-play, can use a phonetic pattern-matching algorithm (answers don't need to be exact), and has many other options. Originally developed by Eric Fears, now maintained by Andy Alt
+site: https://github.com/andy5995/blitzed-irc-trivia/wiki
+tags:
+- c
+- c++
+- IRC
+- games
+- trivia
+- cli
+- chat
+upforgrabs:
+  name: Blitzed-Irc-Trivia
```

Comment:
I wonder if I can have the link point to the "Contribution" guidelines page? From there I link to the issues. The last time I had a project listed here, people would ask how to get started, and I'd always have to point them to my "Contributions" page.